src/main/fc/fc_msp.c

```diff
@@ -1186,6 +1187,37 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
         break;
 
+    case MSP_VTX_CONFIG:
+#if defined(VTX_COMMON)
+        {
+            uint8_t deviceType = vtxCommonGetDeviceType();
+            if (deviceType != VTXDEV_UNKNOWN) {
+
+                uint8_t band=0, channel=0;
+                vtxCommonGetBandChan(&band,&channel);
+                
+                uint8_t powerIdx=0; // debug
+                vtxCommonGetPowerIndex(&powerIdx);
+                
+                uint8_t pitmode=0;
+                vtxCommonGetPitmode(&pitmode);
+                
+                sbufWriteU8(dst, deviceType);
+                sbufWriteU8(dst, band);
+                sbufWriteU8(dst, channel);
+                sbufWriteU8(dst, powerIdx);
+                sbufWriteU8(dst, pitmode);
+                // future extensions here...
+            }
+            else {
+                sbufWriteU8(dst, VTXDEV_UNKNOWN); // no VTX detected
+            }
+        }
+#else
+        sbufWriteU8(dst, 0); // VTX not supported
```

Comment:
It is a better way to indicate that VTX is not supported by just not supporting the command entirely, and let MSP return an unknown command.

This also makes handling on the client side easier, as it's the same pattern that is shown by firmware that was built before the command was defined.